pkg/trace/api/api.go

```diff
@@ -178,6 +179,8 @@ func (r *HTTPReceiver) attachDebugHandlers(mux *http.ServeMux) {
 		pprof.Handler("block").ServeHTTP(w, r)
 		runtime.SetBlockProfileRate(0)
 	})
+
+	mux.HandleFunc("/debug/vars", expvar.Handler().ServeHTTP)
```

Comment:
You can do this:
```suggestion
	mux.Handle("/debug/vars", expvar.Handler())
```

Reply:
Made the change myself in the hope that it gets included in the currently being released release.